descartes_planner/include/descartes_planner/dense_planner.h

```diff
@@ -23,15 +23,15 @@ class DensePlanner: public descartes_core::PathPlannerBase
 
   virtual bool initialize(descartes_core::RobotModelConstPtr &model);
   virtual bool setConfig(const descartes_core::PlannerConfig& config);
-  virtual void getConfig(descartes_core::PlannerConfig& config);
+  virtual void getConfig(descartes_core::PlannerConfig& config) const;
   virtual bool planPath(const std::vector<descartes_core::TrajectoryPtPtr>& traj);
```

Comment:
You could still use it in a non const-qualified way. planPath is a big, expensive call and mutates basically all of the state in the object. Calling it changes the fundamental nature of the planner.